### ISSUE TITLE
Keyboard navigation (up/down/enter) doesn't work with Angular 1.5 and 0.14.9

### ISSUE THREAD

**Issue by JoeFirebaugh** [angular-ui/ui-select#1488]
Currently our project is running fine with Angular 1.4.7 and UI-Select 0.13.1...   

However, I'm being tasked to upgrade our project to use Angular 1.5.

Upon doing that I get the ng.animate problem that's been reported a few times.   So I update to UI-Select 0.14.9 and the keyboard navigation (up/down/enter) no longer works when the options are open.  I also see people reporting that in the Issues, with a few folks contributing Pull Requests... So, I suspect my solution is in one or more of the Pull-Requests... Is there a better fork of this project somewhere?  Admittedly I'm a newbie in customizing a GitHub repo with specific PR's...

**Comment by JoeFirebaugh** [comment]
I'm just going to close this for now, and maybe reopen it if I can create a specific plunkr.   As I've found one Angular 1.5.0 plunkr that does show the keyboard navigation working.
